lang/en/tool_heartbeat.php

```diff
@@ -30,6 +30,13 @@
 $string['progresshelp'] = 'This tests that all the various output buffers in the entire stack are corrent including but not limited to php, ob, gzip/deflat, varnish, nginx etc';
 $string['testwarning'] = 'Fake a warning';
 $string['testerror'] = 'Fake a critical';
+$string['ipconfig'] = 'IP Config';
```

Comment:
this is not your fault at all, just an fyi but generally it's nice if the language string files are sorted by key.

If you find that all your new keys are spread out it's probably a sign that they have inconsistent names and maybe should have a common prefix to group them nicely.